### DISCUSSION TITLE
ADO LDSENK02F

### DISCUSSION THREAD

**Discussion by navycrow** [Koenkk/zigbee2mqtt#12048] [Devices]
hello,
I bought this plug also known under LEXMAN trademark.
It's working fine. Could you add power outage memory (ON/OFF works for sure on genOnOff : startupOnOff but don't know about RESTORE)
haElectricalMeasurement:rmsVoltage and haElectricalMeasurement:rmsCurrent can be read too but stay to 0 ?
thx

**Comment by economistgame** [comment]
Hi, I have one model similar as  LDSENK02F called LDSENK02S that has the same aspect but nowadays I am checking if I can meter but appears as you told 0. I understand that maybe use another topic. But I do not have it the official ENKO gateway thus we need to investigate the protocol, if I found to my LDSENK02S, I will return again to send you an "external configuration" in order to check if works
